### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by sonyco-4u**
In this project I did the following
1. updated style.css
2. updated index.html
3. Add media queries to make sure that the website displays correctly on desktop screens.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,1115 @@
...
+  .lang-text {
+    text-align: center;
+  }
+
+  .about-grid {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    padding-bottom: 100px;
+  }
```
Comment (Wes-Isaac) [inline styles.css:748]: - [ ] great job using flex to layout this content, however, the **spacing(padding, margin)** is different than the one specified on the design, kindly fix that.

### Your website
Screen Shot 2022-09-03 at 9 58 40 AM

### Figma's design
Screen Shot 2022-09-03 at 10 00 57 AM

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,1115 @@
...
+  margin: 30px;
+  gap: 20px;
+}
+
+.form-text {
+  border: 1px solid white;
+  border-radius: 8px;
+  height: 48px;
+  padding: 15px;
+  font-family: Poppins, sans-serif;
```
Comment (Wes-Isaac) [inline styles.css:608]: - [ ] Great job implementing button interactions, kindly reduce the border-radius of the input elements to match the design.

### Your website
Screen Shot 2022-09-03 at 10 02 11 AM

### Figma's design
Screen Shot 2022-09-03 at 10 04 27 AM

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,1115 @@
...
+.dcanopy2_face,
+.dcanopy2_facebook,
+.desktop-image {
+  display: none;
+}
+
+@media (min-width: 992px) {
+  .sidebar {
+    display: none;
+  }
```
Comment (Wes-Isaac) [inline styles.css:672]: - [ ] Great job implementing the styles with a mobile first approach, however it's specified that we should use `min-width: 768px`, kindly change that.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @sonyco-4u, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README 👀.
- Good commit message ✔️.
- No Linter errors ✅.
- Follow Github flow ✔️.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ichala — APPROVED** [review]
Hi @sonyco-4u ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Good Design✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,1115 @@
...
+    margin: 0 200px;
+  }
+
+  .project-card {
+    flex-direction: row;
+    gap: 20px;
+    border: 1px solid #dfe1e6;
+    width: 1156px;
+    height: 496px;
+  }
```
[the remark this answers is not in the document]
Comment (sonyco-4u) [reply styles.css:925]: I have corrected all the erros thanks

File: styles.css (hunk 1 above)
Comment (sonyco-4u) [reply styles.css]: thanks I have made the corrections

File: styles.css (hunk 2 above)
Comment (sonyco-4u) [reply styles.css]: thanks I have made the corrections

File: styles.css (hunk 3 above)
Comment (sonyco-4u) [reply styles.css]: thanks I have made the corrections

**Pull request merged**